git/remote.py

```diff
@@ -1068,7 +1068,7 @@ def fetch(
             Git.check_unsafe_options(options=list(kwargs.keys()), unsafe_options=self.unsafe_git_fetch_options)
 
         proc = self.repo.git.fetch(
-            "--", self, *args, as_process=True, with_stdout=False, universal_newlines=True, v=verbose, **kwargs
+            "--", self, *args, as_process=True, with_stdout=False, v=verbose, **kwargs
```

Comment:
Since CI is now failing in a seemingly unrelated test (related to submodules) on Unix platforms, maybe this means that something is fixed now and the the expectation in the submodule test is wrong?
On Windows it doesn't run, which is why it's not failing there.